src/zenml/cli/artifact.py

```diff
@@ -281,3 +296,28 @@ def prune_artifacts(
         except Exception as e:
             cli_utils.error(str(e))
     cli_utils.declare("All unused artifacts and artifact versions deleted.")
+
+
+def _artifact_version_to_print(
+    artifact_version: ArtifactVersionResponse,
+) -> Dict[str, Any]:
+    return {
+        "id": artifact_version.id,
+        "name": artifact_version.artifact.name,
+        "version": artifact_version.version,
+        "uri": artifact_version.uri,
+        "type": artifact_version.type,
+        "materializer": artifact_version.materializer,
+        "data_type": artifact_version.data_type,
+        "tags": [t.name for t in artifact_version.tags],
+    }
+
+
+def _artifact_to_print(
+    artifact_version: ArtifactResponse,
+) -> Dict[str, Any]:
```

Comment:
It is already `Union[str,UUID,List[str]]`. I can update this, but I don't care much cause it is printer utility for CLI. Keeping it as Any and letting `__repr__` do the rest is easier here.
For bigger functions like `_artifact_version_to_print` it will be a nigtmare 🙂  See 6+ types already.